src/Controls/src/Core/Handlers/Items/CarouselViewHandler.Windows.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Maui.Controls.Platform;
+using Microsoft.Maui.Handlers;
+using Microsoft.UI.Xaml.Controls;
+using UWPApp = Microsoft.UI.Xaml.Application;
+using UWPDataTemplate = Microsoft.UI.Xaml.DataTemplate;
+using WScrollBarVisibility = Microsoft.UI.Xaml.Controls.ScrollBarVisibility;
+using WScrollMode = Microsoft.UI.Xaml.Controls.ScrollMode;
+using WSnapPointsAlignment = Microsoft.UI.Xaml.Controls.Primitives.SnapPointsAlignment;
+using WSnapPointsType = Microsoft.UI.Xaml.Controls.SnapPointsType;
+
+namespace Microsoft.Maui.Controls.Handlers.Items
+{
+	public partial class CarouselViewHandler : ItemsViewHandler<CarouselView>
+	{
+		protected override IItemsLayout Layout => VirtualView?.ItemsLayout;
+		LinearItemsLayout CarouselItemsLayout => VirtualView?.ItemsLayout;
+
+		UWPDataTemplate CarouselItemsViewTemplate => (UWPDataTemplate)UWPApp.Current.Resources["CarouselItemsViewDefaultTemplate"];
+
+		public static void MapIsSwipeEnabled(CarouselViewHandler handler, CarouselView carouselView)
```

Comment:
Could include the `[MissingMapper]` flag?. Later is easier to find pending stuff.